synapse/server.py

```diff
@@ -256,6 +257,14 @@ def get_instance_id(self):
         """
         return self.instance_id
 
+    def get_instance_name(self) -> str:
```

Comment:
it'd be nice to actually use this in the various places that currently use `config.worker_name` (I found a few dotted around), rather than having two ways of doing the same thing.